plume-db/src/main/java/com/coreoz/plume/db/pagination/Slice.java

```diff
@@ -0,0 +1,39 @@
+package com.coreoz.plume.db.pagination;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ * Represents a portion (or slice) of a larger dataset.
+ *
+ * @param items the items in the slice
+ * @param hasMore boolean set to true if there is another slice after
```

Comment:
`@param hasMore boolean set to true if another slice is available after this one`